R/utils.R

```diff
@@ -1,44 +1,41 @@
-#' Removes curly brackets from string
+#' Removes leading and trailing curly brackets from character
+#' string and removes indentation of remaining contents
 #'
 #' @description `r lifecycle::badge("stable")`
 #' @param x (`character`)\cr
-#'  string with code chunk
 #'
-#' @return The string without curly braces
+#' @return Character string without curly braces
```

Comment:
```suggestion
#' @return `character(1)` string without curly braces.
```

Reply:
so it doesn't always return character(1) - but I have removed the upper case "C"